common/lazy-load-assemblies-wasm/LazyLoadTelerikComponents/Client/Pages/Counter.razor

```diff
@@ -0,0 +1,16 @@
+﻿@page "/counter"
```

Comment:
I think this page can be removed - this put the focus on 2 options only - lazy loaded page and non-lazy-loaded page.